src/device-manager/python/weave-device-mgr.py

```diff
@@ -1284,6 +1289,7 @@ def do_createthreadnetwork(self, line):
         optParser = OptionParser(usage=optparse.SUPPRESS_USAGE, option_class=ExtendedOption)
         optParser.add_option("-n", "--name", action="store", dest="threadNetworkName", type="string")
         optParser.add_option("-k", "--key", action="store", dest="threadNetworkKey", type="string")
+        optParser.add_option("-s", "--pskc", action="store", dest="threadPskc", type="string")
```

Comment:
use `-C` to keep consistent with *otbr-commissioner*? And unify the variable name to be `threadPSKc`?

Reply:
I'm not sure whether we tend to use capital letters in weave-device-manger.

Reply:
@robszewczyk Thoughts?